src/winfspy/memfs.py

```diff
@@ -188,6 +189,26 @@ def __init__(self, volume_label):
         self._entries = {self._root_path: self._root_obj}
         self._thread_lock = threading.Lock()
 
+        # For debugging purposes
+        # self._add_file("C:\\Users\\User\\Downloads\\sample.pdf")
+        # self._add_dir("abcdef")
+
+    def _add_dir(self, path):
+        path = self._root_path / path
+        obj = FolderObj(
+            path, FILE_ATTRIBUTE.FILE_ATTRIBUTE_DIRECTORY, self._root_obj.security_descriptor,
+        )
+        self._entries[path] = obj
+
+    def _add_file(self, file_path):
```

Comment:
The `_add_dir` creates a new empty directory where `_add_file` import an existing file, so having the same naming pattern for them is confusing.
I would rename them `_create_folder` (using folder instead of dir given we have `FolderObj`) and `_import_file`

Reply:
:+1: